sale_saleor/helpers.py

```diff
@@ -3,7 +3,9 @@
 import time
 
 from bs4 import BeautifulSoup, NavigableString
+from django.utils.text import slugify
```

Comment:
@Kimkhoi3010 because we only need this tiny function in utils ([link](https://github.com/django/django/blob/main/django/utils/text.py#L468)), I think we can copy/paste (with a note borrow from django slugify) instead install whole django package